frontend/src/components/nodes/__test__/NodeRow.test.tsx

```diff
@@ -17,8 +17,8 @@ describe("<NodeRow />", () => {
             agentVersion: "0.4.5",
             agentBuild: "6a527109-modified",
             status: "NoSync",
-            latitude: 38.7936,
-            longitude: -90.7854,
+            latitude: "38.7936",
+            longitude: "-90.7854",
```

Comment:
As far as I remember we are not using this params anywhere but still is it OK that we had numbers and now strings?

Reply:
We don't use them and I just complied with what `pg-node` resulted in queries.